common/tls/src/main/java/io/helidon/common/tls/ConfiguredTlsManager.java

```diff
@@ -91,301 +87,154 @@ public String type() {
     }
 
     @Override // TlsManager
-    public void decorate(TlsConfig.BuilderBase<?, ?> target) {
-        sslParameters(target);
-        sslContext(target, this::keyManager, this::trustManager);
+    public SSLContext sslContext() {
+        return sslContext;
     }
 
     @Override // TlsManager
-    public void init(Tls tls) {
-        this.enabled = tls.enabled();
+    public void init(TlsConfig tlsConfig) {
+        sslContext(tlsConfig);
```

Comment:
If SSLContext is coming out of the builder, do you want to remove it from the blueprint?  I had it so the manager->decorator was building the SSLContext that would get into the finalized config object.  I thought we wanted the SSLContext to be a long-lived instance, where only the underlying key and trust managers would mutate.

Reply:
From which blueprint? We can still set an explicit `SSLContext` on the builder of `TlsConfig` (and it is honored by the `ConfiguredTlsManager`). 
"SSLContext to be a long -lived instance": That is correct. `init(TlsConfig)` is called exactly once for the lifetime of a `Tls` instance, and the `SSLContext` instance is never changed after this method is invoked. I have also modified the OCI manager to honor this approach (it inits SSLContext just once on init, and calls `reload()` on sub-sequent changes to the certificates/keys